pages/profile/v2/index.tsx

```diff
@@ -32,30 +33,31 @@ export default function Profile(props) {
   return (
     <div className="flex flex-col p-4 m-4 overflow-auto bg-scroll">
```

Comment:
space-y-[] here to space the expandable rows

Reply:
perfect